### ISSUE TITLE
Improve instructions for M1 Mac Unit Testing

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#10626]
We probably want to follow up with instructions for M1 macs if `.init()` fails, but I'll leave that as lower priority than getting myself unblocked.

_Originally posted by @david-allison in https://github.com/ankidroid/Anki-Android/issues/10625#issuecomment-1079182281_

**Comment by david-allison** [comment]
Reference: 

https://github.com/ankidroid/Anki-Android/pull/10625/files#diff-e78b257118f0dec12fedc006951194c5a7b3dde32b443c52c31542aa1a6976d6R108

Add a try...catch for the error listed in https://github.com/ankidroid/Anki-Android-Backend/issues/164

And give them instructions for how to solve this. See: https://github.com/ankidroid/Anki-Android/pull/10625

**Comment by david-allison** [comment]
Sounds good!

**Comment by parthn2** [comment]
HIi! Can you please tell how to give the instructions. Is it like writing instructions in documentation and then redircting them to it from the terminal?

**Comment by david-allison** [comment]
> HIi! Can you please tell how to give the instructions. Is it like writing instructions in documentation and then telling the position for solving error the terminal? And just to confirm is the error for m1 gone as all the test are passing as written here [#10625 (comment)](https://github.com/ankidroid/Anki-Android/pull/10625#issuecomment-1079188502)

Throw an exception with appropriate information.

The tests for M1s only pass if the provided environment variables are set. Once the variables are set, all tests pass.
